openeo/rest/connection.py

```diff
@@ -1092,20 +1093,20 @@ def list_files(self):
         """
         Lists all files that the logged in user uploaded.
 
-        :return: file_list: List of the user uploaded files.
+        :return: file_list: List of the user-uploaded files.
         """
 
         files = self.get('/files', expected_status=200).json()['files']
+        files = [UserFile(file['path'], self, file) for file in files]
         return VisualList("data-table", data=files, parameters={'columns': 'files'})
 
-    def create_file(self, path):
+    def get_file(self, path) -> UserFile:
         """
-        Creates virtual file
+        Gets a (virtual) file for the user workspace
 
-        :return: file object.
+        :return: UserFile object.
         """
-        # No endpoint just returns a file object.
-        raise NotImplementedError()
+        return UserFile(path, self)
```

Comment:
```suggestion
        return UserFile(path, connection=self)
```
to make it look less weird